lib/src/persist/adapter.dart

```diff
@@ -1,40 +1,41 @@
-import 'package:casbin/src/model/model.dart';
+// Copyright 2018-2021 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import '../model/model.dart';
 
 /// Interface for Casbin adapters.
 abstract class Adapter {
   /// Loads all policy rules from the storage.
-  ///
-  /// [model] is the model.
-  void loadPolicy(Model model);
+  Future<void> loadPolicy(Model model);
```

Comment:
Don't remove the comments.

Reply:
The comments are copied from Go and TS implementation.
Some modification was made because dart doesn't suggest comment start with function name and also tagged document for each paramenter. Especially meaningless doc like "[model] is the model".

See also: https://dart.dev/guides/language/effective-dart/documentation#do-use-prose-to-explain-parameters-return-values-and-exceptions

Reply:
Of course the documents still can be improved. I prefer to do them together with other documents (Milestone 6).

Reply:
Removing the params from the comments do more harm than good IMHO, you can refactor them into prose but I think it's better to have them until then.

Reply:
Ok, I will try to refactor them into prose.